src/Node/GethNode.ts

```diff
@@ -128,6 +140,26 @@ export default class GethNode extends Node {
     return args;
   }
 
+  private devGethArgs(chain): string[] {
+    let args = this.getDefaultDockerArgs();
+    let volume = '';
+
+    if (chain === 'dev' || chain === 'origin') {
```

Comment:
We can have  a single util function to identify aux and origin dev chains.
e.g. isAuxDevChain, isOriginDevChain.

There are multiple places these conditions are being used. This will be cleaner ⛱